source/System.h

```diff
@@ -137,6 +137,8 @@ class System {
 	bool HasShipyard() const;
 	// Check whether you can buy or sell ship outfits in this system.
 	bool HasOutfitter() const;
+	// Get the system's invisible fence radius.
+	double InvisibleFenceRadius() const;
```

Comment:
I would move this up to above the jump range/below the asteroid belts functions.

Reply:
It's not moved; :| :| :| :|